react/components/organisms/chat/chat.js

```diff
@@ -300,8 +300,8 @@ const styles = StyleSheet.create({
         marginTop: "40%"
     },
     noMessagesImage: {
-        width: 150,
-        height: 150
+        height: 150,
+        width: "100%"
     },
     noMessagesText: {
         color: "#57626e",

```

Comment:
add a margin so that there's a small separation between text and the image
```suggestion
    noMessagesText: {
        margin: 20,
        color: "#57626e",

```